Automattic/woocommerce-payments
# fix: add confirmation modal when disabling WooPayments

#### Changes proposed in this Pull Request









Discussed here: p1708097977644219-slack-CGGCLBN58

Showing a confirmation modal when the merchant decides to disable WooPayments.
This is the same dialog displayed in the main WC settings page.
![2024-02-19 10 36 04](https://github.com/Automattic/woocommerce-payments/assets/273592/8e9886cd-1c10-4946-8dd0-68b99028379a)


#### Testing instructions





- Go to the WooPayments settings page http://wcpay.test/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Click on the checkbox to enable/disable WooPayments
- When disabling, a confirmation modal is displayed. Confirming on the modal will disable WooPayments.
- When enabling, no confirmation modal is displayed

-------------------


**Post merge**
